src/lando/ui/jinja2/partials/job.html

```diff
@@ -29,36 +29,6 @@
       {%- endfor %}
     </p>
     {% endif %}
-    {% if job.error_breakdown %}
```

Comment:
It's unclear why this is moved. Were there any functional changes here? It's hard to tell from the diff. 😅

Reply:
Ah, good point. This was to show the suggestion to rebase even when the `error_breakdown` is missing, due to the bug getting fixed here. But there are other errors, which aren't fixable by rebasing, so showing that message is moot. More so if we fix the bug that made it desirable to be shown too.